cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
+      "links": [
+        {
+          "description": "All JADE runs",
+          "recid": "FIXME"
+        }
+      ]
+    },
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "26050"
+    },
+    "collections": [
+      "JADE-Primary-Datasets"
+    ],
+    "collision_information": {
+      "energy": "12GeV",
+      "type": "e+e-"
+    },
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 7017835,
+      "number_files": 2861,
+      "size": 2614994979
+    },
+    "doi": "10.7483/OPENDATA.9S5F.BY3B",
+    "experiment": [
+      "JADE"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:44e5147f",
+        "size": 1575978,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/jade/tapes_index.json"
+      }
+    ],
+    "keywords": [
+      "e+e- physics"
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "<p> During the experiment, the raw data was written to the tapes.</p>"
```

Comment:
Any interesting links to point to for the methodology part? Perhaps a link to some JADE computing notes describing the process?